### ISSUE TITLE
Visual improvements for command input

### ISSUE THREAD

**Issue by andrechen77** [NetLogo-Mobile/CodeMirror-NetLogo#110]
For all of the "one-line" [ParseModes](https://github.com/NetLogo-Mobile/CodeMirror-NetLogo/blob/12a1edaaa2ac74d58f78bbea1a26926ca3179dc6/src/editor-config.ts#L47-L58):
- the editor shouldn't have line numbers 
- the editor should have a fixed height (without a vertical scroll bar)
- overflowing the text shouldn't create a horizontal scroll bar, instead just scrolling to the right.

Since these visual changes don't really have to do with parsing, I'm not sure if this would require a name change of the `ParseMode` option or if it would require a whole new option entirely (I'm leaning toward combining the options).

**Comment by CIVITAS-John** [comment]
Let's use the `OneLine` option which makes more sense.

**Comment by CIVITAS-John** [comment]
```
.command-line .command-input .cm-editor {
	flex: 1 1 auto;
	height: auto;
	font-size: 1em;
	max-height: 2rem;
	max-width: calc(100% - 2rem);
	box-sizing: border-box;
}

.command-line .command-input .cm-editor .cm-gutters {
	display: none;
}

.command-line .command-input .cm-editor .cm-scroller {
	line-height: 1.5;
	overflow-x: scroll;
	overflow-y: hidden;
}

.command-line .command-input .cm-editor {
	flex: 1 1 auto;
	height: auto;
	font-size: 1em;
	max-height: 2rem;
	max-width: calc(100% - 2rem);
	box-sizing: border-box;
}

.command-line .command-input .cm-editor span.cm-placeholder .active {
	color: #cc2200;
}

.command-line .command-input .cm-editor .cm-scroller::-webkit-scrollbar {
	display: none;
}

.command-line .command-input .cm-editor .cm-activeLine {
	background: transparent;
}
```

This is what I used in [TU-Editor](https://github.com/NetLogo-Mobile/TU-Editor). We may be able to put it inside a custom theme when the OneLine option is on. @andrechen77 Do you think we need to do that? If so, can @coloshword help? Thanks.

**Comment by andrechen77** [comment]
Works like a charm as a drop-in solution. However, it still feels like this should be the responsibility of the CodeMirror-NetLogo module, doesn't it? I mean, we're reaching into class like `cm-editor` and `cm-scroller` and all these gory details that should have been abstracted away.

In this sense, I think we can consider the issue temporarily resolved (for the purpose of pretty demos and stuff), but it should remain open since the ideal solution is as you said: apply these styles when the OneLine option is on, *inside* CodeMirror-NetLogo.

P.S. did you intend to repeat the ruleset for `.command-line .command-input .cm-editor`?

**Comment by CIVITAS-John** [comment]
No, that was an oversight. Thank you. @coloshword we can work to integrate some of this when you don't have other responsibilities
